pkgs/misc/vim-plugins/default.nix

```diff
@@ -309,7 +309,7 @@ self = generated // (with generated; {
     '';
   });
 
-  yankring = yankring.overrideAttrs(old: {
+  YankRing-vim = YankRing-vim.overrideAttrs(old: {
```

Comment:
Do you have an idea how to rewrite this so it would fail to evaluate also when `allowAliases` has its default value? Otherwise we will see this also in future.

Reply:
Yes, we can do something like is done with `top-level/aliases.nix`. I will look into this, since I want to use this configuration option with r-ryantm/nixpkgs-update and it would be nice to not have to fix it all the time.

Reply:
@Mic92 Another option would be to make a vim-specific alias configuration: config.allowVimPluginAliases then vim plugin people can break their own evaluation whenever they want.

Reply:
@ryantm sounds like bandaid.